tikv-client/src/main/java/com/pingcap/tikv/operation/iterator/DAGIterator.java

```diff
@@ -143,7 +143,11 @@ private boolean readNextRegionChunks() {
     }
 
     taskIndex++;
-    return advanceNextResponse();
+    if (advanceNextResponse()) {
```

Comment:
Please add some comments. This is very hard to catch.

Reply:
good idea